.travis.yml

```diff
@@ -119,14 +121,8 @@ before_script:
 
   # GCE login
   - echo $GCE_KEY > ./bin/account_file.json
-
-  - >
-      gcloud auth activate-service-account
-      12202776487-compute@developer.gserviceaccount.com
-      --key-file=$GCE_ACCOUNT_FILE_PATH
-  - >
-      gcloud auth login 12202776487-compute@developer.gserviceaccount.com
-      --project phenomenal-1145 -q
+  - gcloud auth activate-service-account 12202776487-compute@developer.gserviceaccount.com --key-file=$GCE_ACCOUNT_FILE_PATH
+  - gcloud auth login 12202776487-compute@developer.gserviceaccount.com --project phenomenal-1145 -q
```

Comment:
Wrote all in one line as it makes it easier to be commented when testing/debugging

Reply:
God point! As long as it is within yamllint constraint should be preferable.